spring-cloud-dataflow-admin-local/pom.xml

```diff
@@ -34,8 +34,8 @@
 				<directory>src/main/resources</directory>
 				<filtering>true</filtering>
 				<includes>
-					<include>META-INF/spring.factories</include>
 					<include>admin.yml</include>
+					<include>META-INF/spring.factories</include>
```

Comment:
as Eric commented here: https://github.com/spring-cloud/spring-cloud-dataflow/commit/40e321827b5cc9937fdc084c7ef7f1d3e6a3707f#commitcomment-15854422 - I don't think the `spring.factories` file should be included here in the first place, should it?